src/languages/en.js

```diff
@@ -567,6 +567,15 @@ export default {
         copy: 'Copy',
         disable: 'Disable',
     },
+    recoveryCodeForm: {
+        error: {
```

Comment:
@MonilBhavsar We need to have Copy verification for all these strings?

Reply:
Can we kindly add the label for text verification

Reply:
@MonilBhavsar Friendly bump

Reply:
Sorry, I missed this. Commented on the issue. Once verified, we can get a spanish translation

Reply:
Translations are confirmed - https://github.com/Expensify/App/issues/22335#issuecomment-1684060434